lib/components/color-select/color-select.module.scss

```diff
@@ -19,7 +19,6 @@
     border: 1px solid var(--border-color);
     border-radius: 50%;
     transition: padding 100ms;
-    cursor: pointer;
```

Comment:
These options are button elements. Buttons don't have `cursor: pointer` by default.

Reply:
Reverted all the cursor changes